nbs/core.ipynb

```diff
@@ -35,7 +35,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #374.        def _get_needed_temporal_cols(self):
</pre><p>Please rename this to <code>_get_needed_exog</code>, since the temporal cols also include the target and the mask  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>